ruma-serde/src/canonical_json/value.rs

```diff
@@ -0,0 +1,244 @@
+use std::{
+    collections::BTreeMap,
+    convert::{TryFrom, TryInto},
+    fmt,
+};
+
+use js_int::Int;
+use serde::de::Error;
```

Comment:
Let's import `Serialize`, `Serializer`, `Deserialize` & `Deserializer` too. No need to be as explicit on use.